packages/component-library/src/components/plmg-slider/test/plmg-slider.e2e.ts

```diff
@@ -50,6 +50,15 @@ describe('plmg-slider sets the range and initial values', () => {
   });
 
   it('sets the value to the min if default value prop is above the allowed range', async () => {
```

Comment:
I think something has gone wrong with your `main` merge since this test name does not relate to the test case.